SAMtags.tex

```diff
@@ -90,6 +90,9 @@ \section{Standard tags}
   {\tt MD} & Z & String for mismatching positions \\
   {\tt MF} & ? & Reserved for backwards compatibility reasons \\
   {\tt MI} & Z & Molecular identifier; a string that uniquely identifies the molecule from which the record was derived \\
+  {\tt MM} & i & Base modifications / methylation  \\
+% {\tt MP} & i & Base modification qualities \\
```

Comment:
is this commented out on purpose?

Reply:
Yes.  MP was the previous definition using phred scale.  I could completely remove it, but kept it incase we ever want to go back to it as it's not 100% clear what people actually want!  ML though I think is more realistic, so that's what I changed to and commented out MP until we're sure it's absolutely not required.